guide/interactions/buttons.md

```diff
@@ -0,0 +1,321 @@
+<div is="discord-messages">
+	<discord-message profile="bot">
+		Pong! (ephemeral + components)
+		<discord-embed
+			slot="embeds"
+			color="#0099ff"
+			title="Some title"
+			url="https://discord.js.org/"
+		>
+			Some description here
+		</discord-embed>
+	</discord-message>
+</div>
+
+Additionally, if you don't want to construct an `ActionRow` every time, you can also pass an array of arrays containing components like this:
+
+```js {7-10,18}
+const { MessageActionRow, MessageButton, MessageEmbed } = require('discord.js');
+
+client.on('interaction', async interaction => {
+	if (!interaction.isCommand()) return;
+
+	if (interaction.commandName === 'ping') {
+		const button = new MessageButton()
+			.setCustomID('primary')
+			.setLabel('primary')
+			.setStyle('PRIMARY');
+
+		const embed = new MessageEmbed()
+			.setColor('#0099ff')
+			.setTitle('Some title')
+			.setURL('https://discord.js.org/')
+			.setDescription('Some description here');
+
+		await interaction.reply({ content: 'Pong!', ephemeral: true, embeds: [embed], components: [[button]] });
+	}
+});
+```
+
+### Disabled buttons
+
+If you want to prevent a button from being used, but not remove it from the message, you can make use of the `setDisabled()` method, which will make it unclickable for everyone:
+
+```js {13}
+const { MessageActionRow, MessageButton } = require('discord.js');
+
+client.on('interaction', async interaction => {
+	if (!interaction.isCommand()) return;
+
+	if (interaction.commandName === 'ping') {
+		const row = new MessageActionRow()
+			.addComponents(
+				new MessageButton()
+					.setCustomID('primary')
+					.setLabel('primary')
+					.setStyle('PRIMARY')
+					.setDisabled(true),
+			);
+
+		await interaction.reply({ content: 'Pong!', components: [row] });
+	}
+});
+```
+
+### Emoji buttons
+
+The label of a button can hold unicode emojis, but if you want to use guild emojis you have to use the `setEmoji()` method like so:
+
+```js {13}
+const { MessageActionRow, MessageButton } = require('discord.js');
+
+client.on('interaction', async interaction => {
+	if (!interaction.isCommand()) return;
+
+	if (interaction.commandName === 'ping') {
+		const row = new MessageActionRow()
+			.addComponents(
+				new MessageButton()
+					.setCustomID('primary')
+					.setLabel('primary')
+					.setStyle('PRIMARY')
+					.setEmoji('123456789012345678'),
+			);
+
+		await interaction.reply({ content: 'Pong!', components: [row] });
+	}
+});
+```
+
+Now you know all there is to building and sending a `MessageButton`! Let's move on to how to receive buttons!
+
+
+## Receiving buttons
+
+Whilst you can receive and handle a `ButtonInteraction` via the interaction event, we recommend you use one of the collectors we'll be covering in the next section. To receive a `ButtonInteraction`, attach an event listener to your client and use the `Interaction#isButton()` type guard to make sure you only receive component interactions.
```

Comment:
I would really not recommend collectors over the global event for all use cases. There are clearly features where collectors are not the right choice, this makes it sound like they're always preferable.